cobc/error.c

```diff
@@ -63,10 +70,80 @@ print_error_prefix (const char *file, int line, const char *prefix)
 	}
 }
 
+/* Display a context around the location of the error/warning, only if
+ * cb_diagnostics_show_caret is true
+
+   Only display two lines before and after. No caret yet for the column as
```

Comment:
My intuition is that it's better to display the file as it is, so that it is easy for the developer to visually match the error location in a terminal with the one in an editor.

Reply:
Totally reasonable

Reply:
My intuition is that it's better to display the file as it is, so that it is easy for the developer to visually match the error location in a terminal with the one in an editor.